src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -39,6 +40,7 @@ public void shooterInit() {
     SmartDashboard.putNumber(SMART_DASHBOARD_TARGET_VELOCITY, targetVelocity);
     SmartDashboard.putString(SMART_DASHBOARD_POSITION, positionOfArm.toString());
     leftShootMotor.setInverted(true); // TODO: Change to Right/Left to invert shooting motor
+    TalonFX.optimizeBusUtilizationForAll(pivotMotor, leftShootMotor, rightShootMotor);
```

Comment:
I think you have to make sure to enable the ones you do care about before calling this method. Unsure if that's why the motors aren't working or not.